### ISSUE TITLE
No highlights by default?

### ISSUE THREAD

**Issue by harrisoncramer** [deckgo/gatsby-remark-highlight-code#40]
When writing out a markdown post using the ``` syntax, the text appears unhighlighted by default. I'm using this plugin in combination with gastby-transformer-remark.

Is it possible to use the above-mentioned syntax inside of a markdown file but have the entirety of the text highlighted?

<img width="733" alt="Screen Shot 2021-06-19 at 2 53 06 PM" src="https://user-images.githubusercontent.com/32515581/122652708-1959e080-d10e-11eb-9de1-93041cf823bd.png">

<img width="492" alt="Screen Shot 2021-06-19 at 2 51 06 PM" src="https://user-images.githubusercontent.com/32515581/122652632-cbdd7380-d10d-11eb-923a-37595fd162b5.png">

It's not one of the options to the plugin, wasn't sure if that's possible to configure somehow.

**Comment by peterpeterparker** [comment]
No sure I understand your question, can you provide a sample repo to make it clearer?

Thanks in advance for the details.

**Comment by peterpeterparker** [comment]
Did not hear anything back since a week, therefore I close the issue / question.

I guess @harrisoncramer you find another solution.

Feel free to ping me with more details if it is not the case, happy to reopen it.

**Comment by rgalanakis** [comment]
If a block does not specify a language for the code block (I'll use `tick` instead of a backtick and `space` instead of a space), like:

```
<tick tick tick>
console.log('hi');
<tick tick tick>
```

or:

```
<space space space space>console.log('hi');
```

there is no highlight by default (the entire contents are muted). You need to use:

```
<tick tick tick>js
console.log('hi');
<tick tick tick>
```

**Comment by peterpeterparker** [comment]
@rgalanakis thank you for the clear explanation. Indeed when no language, it should fallback on default aka javascript.

Would you like to give a try to solve the issue?

**Comment by rgalanakis** [comment]
Hi @peterpeterparker , since I found a workaround I doubt I will find the time to do this 😆 
One of my coworkers, @ernes7, could probably take this on if you aren't going to have time for it.

**Comment by peterpeterparker** [comment]
Thanks for the feedback @rgalanakis.

I have no time for it in next weeks.

If you coworker can tackle it that would be for sure awesome.

**Comment by peterpeterparker** [comment]
Thanks everybody for the information and @joshuamorony for the PR

It should now be solved and the fix has been released to npm (`v3.0.1`)
